### PR TITLE
[EDUCATOR-2768] Revert 3rd party transcripts custom workflow

### PR DESCRIPTION
**Pull request by Qubad786**
[EDUCATOR-2768](https://openedx.atlassian.net/browse/EDUCATOR-2768)

For reference:
This reverts: https://github.com/edx/edx-video-pipeline/pull/90

### CODE REVIEW

**Comment by Qubad786** [comment]
@mushtaqak @muhammad-ammar please have a look.

File: static_config.yaml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 ---
 # This configuration should only have static settings.
 
-# 3PlayMedia FTP host for old workflow
-threeplay_ftphost: ftp.3playmedia.com
```
Comment (muhammad-ammar) [inline static_config.yaml]: @Qubad786 do we also need to remove this from veda-secure?

Comment (Qubad786) [reply static_config.yaml]: Yes.

Comment (muhammad-ammar) [reply static_config.yaml]: we also need to remove this from `instance_config.yaml` in code.

Comment (Qubad786) [reply static_config.yaml]: Done!

**Pull request merged**